packages/atoms/src/Modal/Modal.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactNode } from 'react';
+import { Modal as RNModal } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { ModalBody } from './ModalBody';
+import { Box } from '../Box';
+
+type ModalProps = {
+    children: ReactNode;
+    isVisible: boolean;
+    onClose: () => void;
+    hasBackArrow: boolean;
+    onBackArrowClick: () => void;
+    title: string;
+};
+
+const modalStyle = prepareNativeStyle(() => ({
+    flex: 1,
+    justifyContent: 'flex-end',
+    backgroundColor: 'rgba(0, 0, 0, 0.3)',
```

Comment:
please verify with Ondra, that this backdrop color will be same in every theme variant, if not it should be added to theme

Reply:
or better `utils.transparentize(0.3, color.black)`

Reply:
[9b8260a](https://github.com/trezor/trezor-suite/pull/5493/commits/9b8260a980265309f9a8900807cb09f4b0429cfc)